render/handler.go

```diff
@@ -14,6 +13,10 @@ import (
 
 //Handler resolves the rendering of a specific pagem with a given model and template name
 func Handler(w http.ResponseWriter, req *http.Request, page interface{}, page2 *model.Page, templateName string, f func()) {
+	cfg, err := config.Get()
```

Comment:
Rather than calling `config.Get()` every time you can pass a config struct into this function. Doing so means not having to deal with the possible error from get config.

Reply:
This is true and is my prefered way of doing it, I did decide this was too large of a scope creep as the original task was "add a feature flag" - was supposed to be a small one that didn't need a ticket. This has already escalated to me doing a rehaul of configs :D. As requested though, I will refactor the 19 handlers that call the render/handler.go

Reply:
@dandwal I think this is worth doing now so happy to have another story added to clean this up.